UI/MVC.js

```diff
@@ -0,0 +1,327 @@
+const users = [{ login: 'janny_9991', password: '123456' }, { login: 'katya', password: 'katya1' }, { login: '2', password: '2' }];
+
+function randomInteger(min, max) {
+  let rand = min - 0.5 + Math.random() * (max - min + 1);
+  rand = Math.round(rand);
+  return rand;
+}
+
+class MVC {
+  constructor(model, view) {
+    this.model = model;
+    this.view = view;
+  }
+
+  showMorePosts() {
+    const button = document.querySelector('.show-more-posts');
+    let count = 0;
+    button.addEventListener('click', () => {
+      count += 10;
+      this.view.showPosts(count);
+    });
+  }
+
+  like(event) {
+    let like;
+    event.target.className === 'far fa-heart fa-lg' ? like = event.target.closest('.like') : like = event.target;
+    const post = like.closest('.post');
+    if (!like.dataset.liked) { like.dataset.liked = '0'; }
+    if (like.dataset.liked === '0') {
+      like.dataset.liked = '1';
```

Comment:
Why do you need attached data here?
Better use something more global (like model) as a 'state of true' to define current state